crates/turbopath/src/absolute_system_path_buf.rs

```diff
@@ -62,6 +62,10 @@ impl AbsoluteSystemPathBuf {
         Ok(AbsoluteSystemPathBuf(system_path))
     }
 
+    pub fn new_unchecked(raw: impl Into<PathBuf>) -> Self {
```

Comment:
I believe @arlyon recommended making these methods unsafe. Should we start doing that?

Reply:
This one appears to have been a bad rebase and isn't used. I think if and when it reappears, we can reassess.